server/s3_delete_object_action.cc

```diff
@@ -125,6 +132,90 @@ void S3DeleteObjectAction::fetch_object_info_failed() {
   s3_log(S3_LOG_DEBUG, "", "%s Exit", __func__);
 }
 
+void S3DeleteObjectAction::delete_handler() {
+  s3_log(S3_LOG_INFO, stripped_request_id, "%s Entry\n", __func__);
+
+  std::string bucket_versioning =
+      bucket_metadata->get_bucket_versioning_status();
```

Comment:
is it required to cache? this could be (bucket_metadata->get_bucket_versioning_status() == "Enabled")

Reply:
Updated